api/grpc/common/common.pb.go

```diff
@@ -84,10 +85,18 @@ type BlobCommitment struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	Commitment       []byte `protobuf:"bytes,1,opt,name=commitment,proto3" json:"commitment,omitempty"`
+	// A commitment to the blob data.
+	Commitment []byte `protobuf:"bytes,1,opt,name=commitment,proto3" json:"commitment,omitempty"`
+	// A commitment to the blob data with G2 SRS, used to work with length_proof
+	// such that the claimed length below is verifiable.
 	LengthCommitment []byte `protobuf:"bytes,2,opt,name=length_commitment,json=lengthCommitment,proto3" json:"length_commitment,omitempty"`
-	LengthProof      []byte `protobuf:"bytes,3,opt,name=length_proof,json=lengthProof,proto3" json:"length_proof,omitempty"`
-	Length           uint32 `protobuf:"varint,4,opt,name=length,proto3" json:"length,omitempty"`
+	// A proof that the degree of the polynomial used to generate the blob commitment is valid.
+	// It is computed such that the coefficient of the polynomial is committing with the G2 SRS
```

Comment:
I'm personally not able to understand this explanation: not sure if I'm just lacking background knowledge, or if this explanation could be further fleshed out

Reply:
This is also beyond me, copied from text suggested by @bxue-l2